src/components/Gameboard.test.js

```diff
@@ -20,4 +20,9 @@ describe('Gameboard', () => {
     expect(board.length).toEqual(5);
     expect(row.length).toEqual(6);
   });
+
+  it('Should not render Winner component', () => {
```

Comment:
```suggestion
  it('Should not render the Winner component by default', () => {
```

Essayes d'avoir des descriptions de tests qui permettent de cerner exactement ce que tu test, ici par exemple, tu dis que le composant ne doit pas être rendered, mais pas pourquoi